Resources/Prototypes/_RMC14/Entities/Objects/Weapons/Throwable/grenades.yml

```diff
@@ -158,6 +158,23 @@
     maxShrapnel: 0
   - type: CMVocalizeTrigger
 
+- type: entity
+  parent: CMGrenadeBase
+  id: RMCGrenadeTraining
+  name: M07 Training Grenade
+  description: A harmless reusable version of the M40 HEDP, used for training. Capable of being loaded into a grenade launcher, or thrown by hand.
+  components:
+  - type: Sprite
+    sprite: _RMC14/Objects/Weapons/Grenades/m07training.rsi
+  - type: SoundOnTrigger
+    sound: /Audio/Effects/snap.ogg
+  - type: Tag
+    tags:
+    - Grenade
+    - LauncherCompatibleGrenade
+    - RMCGrenadeTraining
```

Comment:
Add the m40 grenade tag on here, it should make it conpatiable with all the launchers in game